packager/modulePaths.js

```diff
@@ -0,0 +1,1079 @@
+    './node_modules/node_modules/mime-db/index.js',
+    './node_modules/node_modules/object-assign/index.js',
+    './node_modules/node_modules/pascalcase/index.js',
+    './node_modules/node_modules/pegjs/lib/compiler/asts.js',
+    './node_modules/node_modules/pegjs/lib/compiler/index.js',
+    './node_modules/node_modules/pegjs/lib/compiler/js.js',
+    './node_modules/node_modules/pegjs/lib/compiler/opcodes.js',
+    './node_modules/node_modules/pegjs/lib/compiler/passes/generate-bytecode.js',
+    './node_modules/node_modules/pegjs/lib/compiler/passes/generate-js.js',
+    './node_modules/node_modules/pegjs/lib/compiler/passes/remove-proxy-rules.js',
+    './node_modules/node_modules/pegjs/lib/compiler/passes/report-duplicate-labels.js',
+    './node_modules/node_modules/pegjs/lib/compiler/passes/report-duplicate-rules.js',
+    './node_modules/node_modules/pegjs/lib/compiler/passes/report-infinite-recursion.js',
+    './node_modules/node_modules/pegjs/lib/compiler/passes/report-infinite-repetition.js',
+    './node_modules/node_modules/pegjs/lib/compiler/passes/report-undefined-rules.js',
+    './node_modules/node_modules/pegjs/lib/compiler/visitor.js',
+    './node_modules/node_modules/pegjs/lib/grammar-error.js',
+    './node_modules/node_modules/pegjs/lib/parser.js',
+    './node_modules/node_modules/pegjs/lib/peg.js',
+    './node_modules/node_modules/pegjs/lib/utils/arrays.js',
+    './node_modules/node_modules/pegjs/lib/utils/classes.js',
+    './node_modules/node_modules/pegjs/lib/utils/objects.js',
+    './node_modules/node_modules/promise/setimmediate/core.js',
+    './node_modules/node_modules/promise/setimmediate/done.js',
+    './node_modules/node_modules/promise/setimmediate/es6-extensions.js',
+    './node_modules/node_modules/promise/setimmediate/rejection-tracking.js',
+    './node_modules/node_modules/prop-types/checkPropTypes.js',
+    './node_modules/node_modules/prop-types/factoryWithTypeCheckers.js',
+    './node_modules/node_modules/prop-types/index.js',
+    './node_modules/node_modules/prop-types/lib/ReactPropTypesSecret.js',
+    './node_modules/node_modules/querystring/decode.js',
+    './node_modules/node_modules/querystring/encode.js',
+    './node_modules/node_modules/querystring/index.js',
+    './node_modules/node_modules/querystringify/index.js',
+    './node_modules/node_modules/raven-js/src/configError.js',
+    './node_modules/node_modules/raven-js/src/console.js',
+    './node_modules/node_modules/raven-js/src/raven.js',
+    './node_modules/node_modules/raven-js/src/singleton.js',
+    './node_modules/node_modules/raven-js/src/utils.js',
+    './node_modules/node_modules/raven-js/vendor/TraceKit/tracekit.js',
+    './node_modules/node_modules/raven-js/vendor/json-stringify-safe/stringify.js',
+    './node_modules/node_modules/raven-js/vendor/md5/md5.js',
+    './node_modules/node_modules/react-deep-force-update/lib/index.js',
+    './node_modules/node_modules/react-devtools-core/build/backend.js',
```

Comment:
Do we need to include `react-devtools-core` in here?

A thought I had was to have a list of files we should be blacklisting from the modulePaths.js instead of manually having to remove each of them and possibly missing one

Reply:
nice catch, I'll remove it from there ;)